dbms/src/Storages/DeltaMerge/DeltaMergeStore.h

```diff
@@ -363,6 +363,7 @@ class DeltaMergeStore : private boost::noncopyable
                            UInt64 max_version,
                            const RSOperatorPtr & filter,
                            const String & tracing_id,
+                           bool is_raw_read = false, // set true when read in fast mode
```

Comment:
update the comment
https://github.com/pingcap/tiflash/blob/cc6017e3077ead3de628e19c7b6abea85e77dba2/dbms/src/Storages/DeltaMerge/DeltaMergeStore.h#L356-L358